sources/packages/web/src/components/students/StudentApplicationsVersion.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <v-table>
+    <thead>
+      <tr>
```

Comment:
I see that if the result set has only one item the divider line is thicker. 

![image](https://github.com/user-attachments/assets/0217722c-e80c-49d3-a3f2-c50b5f1b8430)

Reply:
I am not sure if it is thicker or just a slightly different gray.
I would prefer not to invest time in it.

Reply:
I can change it to a `v-data-table`, but I am not sure if it would be worth it.